dapps/marketplace/src/utils/makeGatewayUrl.js

```diff
@@ -0,0 +1,17 @@
+/**
```

Comment:
As opposed to duplicating code, how about importing the @origin/ipfs package in the dapp and using that function from there ?